src/utils/webrtc/MediaDevicesManager.js

```diff
@@ -83,6 +91,15 @@ export default function MediaDevicesManager() {
 	this._tracks = []
 
 	this._updateDevicesBound = this._updateDevices.bind(this)
+
+	this._pendingEnumerateDevicesPromise = null
+
+	if (localStorage.getItem('audioInputId') === LOCAL_STORAGE_NULL_DEVICE_ID) {
```

Comment:
Please use the storage from `../../services/BrowserStorage.js`

Reply:
Ugh, right :facepalm: Done.